src/index.html

```diff
@@ -1,13 +1,724 @@
+                    <div class="order-button__text">
+                      Order
+                    </div>
+                  </a>
+                </article>
+              </div>
+
+              <div class="
+                grid__item--tablet-5-8
+                grid__item--big-tablet-6-8
+                grid__item--desktop-6-8"
+              >
+                <article class="product__card-container">
+                  <a href="#" class="product__card-image-container">
+                    <img
+                      src="./images/photo-product-4.jpg"
+                      alt="product-card"
+                      class="product__card-image"
+                    >
+                  </a>
+
+                  <div class="product__card-description-container">
+                    <p class="product__card-description-title">
+                      Cakes
+                    </p>
+
+                    <p class="product__card-description-text">
+                      Oatmeal cookies with golden raisins, orange zest and chocolate chips
+                    </p>
+                  </div>
+
+                  <a href="#" class="order-button order-button--product">
+                    <div class="icon icon--order-button"></div>
+                    <div class="order-button__text">
+                      Order
+                    </div>
+                  </a>
+                </article>
+              </div>
+            </div>
+          </div>
+
+          <div class="product__card-block">
+            <div class="grid grid--screen">
+              <div class="
+                grid__item--tablet-1-4
+                grid__item--big-tablet-1-3
+                grid__item--desktop-1-3"
+              >
+                <article class="product__card-container">
+                  <a href="#" class="product__card-image-container">
+                    <img
+                      src="./images/photo-product-5.jpg"
+                      alt="product-card"
+                      class="product__card-image"
+                    >
+                  </a>
+
+                  <div class="product__card-description-container">
+                    <p class="product__card-description-title">
+                      Cakes
+                    </p>
+
+                    <p class="product__card-description-text">
+                      Oatmeal cookies with golden raisins, orange zest and chocolate chips
+                    </p>
+                  </div>
+
+                  <a href="#" class="order-button order-button--product">
+                    <div class="icon icon--order-button"></div>
+                    <div class="order-button__text">
+                      Order
+                    </div>
+                  </a>
+                </article>
+              </div>
+
+              <div class="
+                grid__item--tablet-5-8
+                grid__item--big-tablet-4-6
+                grid__item--desktop-4-6"
+              >
```

Comment:
Consider it will be better
```suggestion
              <div
                class="
                  grid__item--tablet-5-8
                  grid__item--big-tablet-4-6
                  grid__item--desktop-4-6
                "
              >
```